forcewindow.cpp

```diff
@@ -39,6 +39,7 @@ void ForceWindow::saveSettings()
 
 // TODO need to check if this is
 // appropriate and correct and working way to handle messages
+// QA: what about const QString& str ? 
```

Comment:
The same reason - copying data for safety, as I think. But I'm open for discussion)